src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.cpp

```diff
@@ -203,6 +203,14 @@ tuples::TaggedTuple<hydro::Tags::Pressure<DataType>> BondiMichel::variables(
                pow(vars.rest_mass_density, polytropic_exponent_)}}};
 }
 
+template <typename DataType>
```

Comment:
From CI it looks like this needs an explicit instantiation at the end of this file.